Sources/ToucanSource/Bundle/SourceBundleRenderer.swift

```diff
@@ -350,9 +350,11 @@ public struct SourceBundleRenderer {
                     markdown: .init(
                         customBlockDirectives: sourceBundle.blockDirectives
                     ),
-                    outline: .init(levels: [2, 3]),
+                    outline: .init(
+                        levels: sourceBundle.config.contentConfigurations.outlineLevels
+                    ),
                     readingTime: .init(
-                        wordsPerMinute: 238
+                        wordsPerMinute: sourceBundle.config.contentConfigurations.wordsPerMinute
                     ),
                     transformerPipeline: pipeline.transformers[
```

Comment:
Please add the paragraph styles as a parameter here and hook them up with the `HTMLVisitor`.